app/javascript/dashboard/components/widgets/conversation/Message.vue

```diff
@@ -70,8 +70,8 @@ export default {
       default: () => ({}),
     },
     inReplyTo: {
-      type: Object,
-      default: () => ({}),
+      type: Promise,
+      default: Promise.resolve({}),
     },
```

Comment:
Passing a Promise as a prop is not a great idea, ideally the parent component should resolve the promise and pass it, or the fetching should happen in the component itself.